### PR TITLE
Add CPod.app v1.28.0

### PR DESCRIPTION
**Pull request by abheekda1**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/cpod.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  url "https://github.com/z-------------/CPod/releases/download/v1.28.0/CPod-#{version}-mac.zip"
+  name "CPod"
+  desc "Simple, beautiful podcast app"
+  homepage "https://github.com/z-------------/CPod"
+
+  app "CPod.app"
+
+  zap trash: [
+    "~/Library/Application Scripts/com.apple.podcasts.MacPodcastsStorageExtension",
+    "~/Library/Containers/com.apple.podcasts.MacPodcastsStorageExtension",
```
Comment (miccal) [inline Casks/cpod.rb:14]: I am assuming these two files are not specific to this `.app`?

**Comment by ADawesomeguy** [comment]
Now that I think about it, it seems you're right. I'll remove the zap and add a custom one not made using the command. I believe CPod has a custom storage directory but that changes depending on the user's preferences. So in reality should I edit the zap or just remove it entirely?

**Comment by miccal** [comment]
> Now that I think about it, it seems you're right. I'll remove the zap and add a custom one not made using the command. I believe CPod has a custom storage directory but that changes depending on the user's preferences. So in reality should I edit the zap or just remove it entirely?

Just edit it, please.

**Comment by neutric** [comment]
The app should be considered unmaintained as per the author, see notes of latest release.

**Comment by SMillerDev** [comment]
Yeah, if it's unmaintained I don't think we want to ship this. 

Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

**Comment by ADawesomeguy** [comment]
Alright, thanks!

**Pull request closed without merging**